server/graphql/v1/mutations/members.js

```diff
@@ -93,3 +95,36 @@ export async function editMembership(_, args, req) {
 
   return member.update(pick(args, ['publicMessage']));
 }
+
+/** A mutation to edit the public message of all matching members. */
+export async function editPublicMessage(_, { FromCollectiveId, CollectiveId, message }, req) {
+  if (!req.remoteUser || !req.remoteUser.isAdmin(FromCollectiveId)) {
+    throw new errors.Unauthorized("You don't have the permission to edit member public message");
+  }
+  const updatedMembers = await models.Member.update(
```

Comment:
From variable names, it's unclear what this function's return is. One would have to check `update`'s documentation to know that it returns an array with the count in first position.

What about the following:

```suggestion
  const [nbUpdated, updatedMembers] = await models.Member.update(
```

Reply:
Great comment @Betree. I totally agree with your comment. I'll make the change. Thanks for the feedback.